src/components/LoginForm/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import { reduxForm, Field } from 'redux-form/immutable';
+import PropTypes from 'prop-types';
+
+import CustomButton from '../../components/common/CustomButton';
+import CustomInput from '../../components/common/CustomInput';
+import { validations, login } from '../../utils/constraints';
+import { black } from '../../constants/styleConstants';
+import styles from './styles';
+
+const LoginForm = ({
+  handleSubmit, error, submitting, signIn
+}) => {
+  return (
+    <View>
```

Comment:
Have you tried this in a small device? Do you think that a `KeyboardAwareView` is necessary here?

Reply:
Yes, you are right, I had tested with a smaller device, but not with the keyboard. I added KeyboardAwareView, thanks!